src/Grandeljay/Availability/Config.php

```diff
@@ -6,30 +6,51 @@ class Config
 {
     private array $config;
 
-    // Ordered list of possible config file locations.
-    private $filepaths = array(
-        '$HOME/.config/discord-availability/config.json',
-        '/etc/discord-availability/config.json',
-    );
-
     public function __construct()
     {
         $this->loadConfig();
     }
 
     private function loadConfig(): void
     {
-        foreach ($this->filepaths as $path) {
-            $path = str_replace('$HOME', getenv('HOME'), $path);
+        $filepathRelative = 'discord-availability/config.json';
+        $filepaths        = array();
+
+        switch (PHP_OS) {
+            case 'WINNT':
+                $filepaths = array(
+                    '$USERPROFILE/.config/' . $filepathRelative,
+                    '$APPDATA/' . $filepathRelative,
+                );
+                break;
+
+            default:
+                $filepaths = array(
+                    '$HOME/.config/' . $filepathRelative,
+                    '/etc/' . $filepathRelative,
+                );
+                break;
+        }
```

Comment:
I figured it was a matter of taste. An `if`, `then`, `else` would work just as well. Although a `switch` statement uses more lines, I prefer it in this case since it seems more flexible in case we would need to handle another OS.